apps/ledger-live-desktop/src/newArch/features/MemoTag/components/MemoTagField.tsx

```diff
@@ -38,28 +41,33 @@ const MemoTagField = ({
   maxMemoLength,
   CaracterCountComponent,
   autoFocus,
+  placeholder,
+  label,
+  tooltipText,
 }: MemoTagFieldProps) => {
```

Comment:
It's a small detailed but MemoTagFieldProps shares the same name with MemoTagFieldProps from `apps/ledger-live-desktop/src/renderer/families/casper/types.ts`. Maybe just calling the type `Props` would make things clearer